src/gui/rss/automatedrssdownloader.h

```diff
@@ -64,17 +69,21 @@ class AutomatedRssDownloader: public QDialog
     ~AutomatedRssDownloader();
     bool isRssDownloaderEnabled() const;
 
+protected:
+    virtual void showEvent(QShowEvent *event);
+    virtual void hideEvent(QHideEvent *event);
```

Comment:
It's been 10+ years since I've used C++ in anger - I'm a little rusty and behind the times ...